### PR TITLE
DOCSP-16529 Clarifies source/destination clusters on Quickstart

### PR DESCRIPTION
**Pull request by kennethdyer**
## Description

Clarifies source and destination clusters on the Quickstart page.

## Staging

[Setup](https://preview-mongodbkennethdyer.gatsbyjs.io/cluster-sync/DOCSP-16529-src-dest-type/quickstart/#setup)

## Jira
[DOCS-16529](https://jira.mongodb.org/browse/DOCS-16529)

## Build

* [2024-01-10](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=659edb2edcc0144a51f2dd18)
* [2024-01-12](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65a17e55dcc0144a51ba48d8)
* [2024-01-16](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65a6e76ddcc0144a51048efb)
* [2024-02-08](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65c53a716b2f17c95a461471)

### CODE REVIEW

File: source/quickstart.txt [hunk 1]
```diff
@@ -25,15 +24,24 @@ the rest of the {+c2c-product-name+} documentation.
 Setup
 -----
 
-{+c2c-product-name+} syncs data between two clusters. To run
-``mongosync``, you will need to: 
+{+c2c-product-name+} syncs data between two clusters. 
+
+- Each cluster can be either a replica set or a sharded cluster.
+
+- Sharded clusters do not require the same number of shards.
```
Comment (mdb-ashley) [inline source/quickstart.txt:31]: [suggestion] This is a little difficult to parse -- consider something like, 

"Sharded clusters are not required to have the same number of shards."

File: source/quickstart.txt [hunk 2]
```diff
@@ -25,15 +24,24 @@ the rest of the {+c2c-product-name+} documentation.
...
+
+- The destination cluster can be up to two versions ahead of the source
+  cluster.  The patch version is not important, so long as it meets the
+  minimum patch version requirements.
```
Comment (mdb-ashley) [inline source/quickstart.txt:35]: [suggestion] Would it make sense to link to the requirements here?

**Review by mdb-ashley — CHANGES_REQUESTED** [review]
This is looking good, Kenneth! I left a couple comments for your consideration.

File: source/quickstart.txt (hunk 1 above)
Comment (kennethdyer) [reply source/quickstart.txt:31]: Fixed

**Review by mdb-ashley — APPROVED** [review]
This LGTM. Thanks, Kenneth!

File: source/quickstart.txt [hunk 3]
```diff
@@ -25,15 +24,24 @@ the rest of the {+c2c-product-name+} documentation.
 Setup
 -----
 
-{+c2c-product-name+} syncs data between two clusters. To run
-``mongosync``, you will need to: 
+{+c2c-product-name+} syncs data between two clusters.
```
[the remark this answers is not in the document]
Comment (kennethdyer) [reply source/quickstart.txt:27]: Fixed

File: source/quickstart.txt [hunk 4]
```diff
@@ -25,15 +24,24 @@ the rest of the {+c2c-product-name+} documentation.
...
 
-{+c2c-product-name+} syncs data between two clusters. To run
-``mongosync``, you will need to: 
+{+c2c-product-name+} syncs data between two clusters. 
+
+- Each cluster can be either a replica set or a sharded cluster.
+
+- Sharded clusters are not required to have the same number of shards.
+
+- The destination cluster can be up to two versions ahead of the source
```
[the remark this answers is not in the document]
Comment (kennethdyer) [reply source/quickstart.txt:33]: Fixed

File: source/quickstart.txt [hunk 5]
```diff
@@ -25,15 +24,24 @@ the rest of the {+c2c-product-name+} documentation.
 Setup
 -----
 
-{+c2c-product-name+} syncs data between two clusters. To run
-``mongosync``, you will need to: 
+{+c2c-product-name+} syncs data between two clusters. 
+
+- Each cluster can be either a replica set or a sharded cluster.
```
[the remark this answers is not in the document]
Comment (kennethdyer) [reply source/quickstart.txt:29]: Fixed

**Review by ajayvijayakumar123 — APPROVED** [review]
LGTM!

**Pull request merged**